CHANGELOG.md

```diff
@@ -230,6 +230,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Enforce an epoch scheduler interval when using SWA ([#6588](https://github.com/PyTorchLightning/pytorch-lightning/pull/6588))
 - Fixed TPU Colab hang issue, post training ([#6816](https://github.com/PyTorchLightning/pytorch-lightning/pull/6816))
 - Fixed a bug where `TensorBoardLogger` would give a warning and not log correctly to a symbolic link `save_dir` ([#6730](https://github.com/PyTorchLightning/pytorch-lightning/pull/6730))
+- Fixed minimum torch version for DDP plugin test ([6906])(https://github.com/PyTorchLightning/pytorch-lightning/pull/6906)
```

Comment:
We don't require CHANGELOG updates for typos, docs, test updates, or internal minor changes/refactorings.

```suggestion
```